### DISCUSSION TITLE
Sponsorship and the supabase.io/oss page

### DISCUSSION THREAD

**Discussion by rmontgomery429** [supabase/supabase#3751] [Questions] [answered]
I was curious about sponsorship and noticed the [supabase.io/oss](supabase.io/oss) page is 404ing right now. Is sponsorship via Github still desirable? If so, is the [supabase.io/oss](supabase.io/oss) page going to come back? Also, how is support and feature request feedback facilitated? Github issues? Slack?

**Comment by inian** [comment, marked as answer]
Thanks for reporting @rmontgomery429. the link should have been supabase.io/docs/oss. Fixed now
